### PR TITLE
4 Update version history

### PR DESCRIPTION
**Pull request by julian-cable**
Updated and renamed release notes as discussed, and removed obsolete files.

### CODE REVIEW

File: RedHatStyleGuide/en-US/rel-notes.xml [hunk 1]
```diff
@@ -3,8 +3,12 @@
 <!ENTITY % BOOK_ENTITIES SYSTEM "Release_Notes.ent">
 %BOOK_ENTITIES;
 ]>
+
 <chapter id="chap-Red_Hat_Technical_Writing_Style_Guide-Release_Notes-Release_5">
 	<title>Release 5</title>
+	 <para>
+		Release notes for 5.1 releases:
```
Comment (daobrien) [inline RedHatStyleGuide/en-US/rel-notes.xml]: s/releases/release/ ?

File: RedHatStyleGuide/en-US/rel-notes.xml [hunk 2]
```diff
@@ -32,5 +36,11 @@
 			<para>Minor edits so the guide itself conforms with its own advice.</para>
 
 	</section>
+	 <section id="sect-Red_Hat_Technical_Writing_Style_Guide-Older_Release_Notes">
+		<title>Earlier Releases than 5.0</title>
```
Comment (daobrien) [inline RedHatStyleGuide/en-US/rel-notes.xml]: "Releases Prior to 5.0"?

File: RedHatStyleGuide/en-US/rel-notes.xml [hunk 3]
```diff
@@ -32,5 +36,11 @@
...
+			<para>For change information for earlier releases than 5.0, see the Git commit history at https://github.com/StyleGuides/WritingStyleGuide/commits/dev.</para>
```
Comment (daobrien) [inline RedHatStyleGuide/en-US/rel-notes.xml]: "earlier releases than 5.0" sounds funny in my head. Is it dialectal?

**Review by daobrien — CHANGES_REQUESTED** [review]
Nit picking mostly. No biggy.

File: RedHatStyleGuide/en-US/rel-notes.xml (hunk 2 above)
Comment (julian-cable) [reply RedHatStyleGuide/en-US/rel-notes.xml]: I thought our standard was to use "earlier" or "later" to refer to releases?

File: RedHatStyleGuide/en-US/rel-notes.xml (hunk 1 above)
Comment (julian-cable) [reply RedHatStyleGuide/en-US/rel-notes.xml]: Agree

File: RedHatStyleGuide/en-US/rel-notes.xml (hunk 2 above)
Comment (daobrien) [reply RedHatStyleGuide/en-US/rel-notes.xml]: Yeah I know what you're saying, but in this construction it just sounds weird. If it were "In release 5.0 or earlier" then sure, no problem. Not a hill to die on in this case IDT.

Comment (julian-cable) [reply RedHatStyleGuide/en-US/rel-notes.xml]: How about "Releases Earlier than 5.0"?

Comment (daobrien) [reply RedHatStyleGuide/en-US/rel-notes.xml]: Are we getting hung up on "Thou shalt use 'earlier' and 'later'" at the expense of creating comprehensible English? 

First words in the Inquirer style book: "Any rule can be overruled by 1) using common sense and 2) trusting your ear."

So, I'll call this approved and rely on your judgement.

**Review by daobrien — APPROVED** [review]

File: RedHatStyleGuide/en-US/rel-notes.xml (hunk 2 above)
Comment (julian-cable) [reply RedHatStyleGuide/en-US/rel-notes.xml]: Applied final wording changes.

**Pull request merged**